File: Baekjoon_Online_Judge/NO_15596/15996.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# -------------------------------------- #
# | Created by happiness96             | #
# | Year 2020                          | #
# | Month 09                           | #
# | Day 02                             | #
# | 정수 N개의 합                       | #
# -------------------------------------- #


def solve(a: list) -> int:
    ans = 0

    for val in a:
        ans += val

    return ans

File: Baekjoon_Online_Judge/NO_15596/15996.cpp
#include <vector>
/*
 -------------------------------------- 
 | Created by happiness96             | 
 | Year 2020                          | 
 | Month 09                           | 
 | Day 02                             | 
 | 정수 N개의 합                       | 
 -------------------------------------- 
*/

long long sum(std::vector<int> &a){
    long long ans = 0;

    for (auto val: a){
        ans += val;
    }

    return ans;
}